hotspots/pharmacophore_extension.py

```diff
@@ -905,7 +905,10 @@ def __init__(self, _feature_def=None):
         super().__init__(_feature_def=_feature_def)
 
     def detect_features(self, crystal):
-        _csv = ChemistryLib.CrystalStructureView_instantiate(crystal._crystal)
+        try:
```

Comment:
We should try to think of a way of removing the need for this as it exposes the internals of the API, but needs must for now.